### PR TITLE
github: add buildx

### PR DESCRIPTION
**Pull request by amarts**
* Currently added another workflow other than already existing 'Release' specific `dockerpush.yml`
* Plan is to run a buildx for every master push, once this is successful, so people can have official images for testing newer features!
* `xxhash` pip install had issues on arm64 architecture.. for now, replaced that with std hashlib. Need to investigate further

* Noticed we can't get 'secrets' used in github actions for `pull_requests` job! only available for operations on master branch like push or tags

* buildx needed to have 'name' field in --output to work properly, wasted more than few hours because ot that!

* fedora:31 images are not present for linux/686, arm/v6, arm/v7, so not having it in build.

Signed-off-by: Amar Tumballi <amar@kadalu.io>

[skip ci]

### CODE REVIEW

**Comment by amarts** [comment]
Overrides #230

**Comment by amarts** [comment]
Finally was able to get `pip install grpcio`  pass... only major issue is, it took ~1hr 20mins to get compiled and installed. Will revert to proper steps and all arch now, and trigger a run, it may take 4hr+ to complete, but would be good to have master tag working on all arch!

**Comment by amarts** [comment]
Please merge with 'Squash and Merge' .... Some tests are good... people can help testing with `kubectl create -f https://raw.githubusercontent.com/kadalu/kadalu/49e304563ed50e76969e77ed4ec8dc568d2fd977/manifests/kadalu-operator-master.yml` on arm64 architecture

**Comment by amarts** [comment]
notice that s390x / ppc64le still threw some issue related to grpcio... So, the first cut only has support for amd64/arm64

File: server/setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def version():
     version=version(),
     packages=["kadalu_quotad"],
     include_package_data=True,
-    install_requires=["requests", "xxhash"],
+    install_requires=["requests", "cython", "xxhash"],
```
Comment (amarts) [inline server/setup.py]: Not required! remove

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,9 @@
 requests
-xxhash
-grpcio
-pyxattr
 kubernetes
 jinja2
 setuptools
 twine
+cython
```
Comment (amarts) [inline requirements.txt]: not required!

File: server/setup.py (hunk 1 above)
Comment (amarts) [reply server/setup.py]: Removed

File: requirements.txt (hunk 2 above)
Comment (amarts) [reply requirements.txt]: Done

**Review by aravindavk — APPROVED** [review]

**Comment by amarts** [comment]
If you want to try now, just run `kubectl create -f https://raw.githubusercontent.com/kadalu/kadalu/master/manifests/kadalu-operator-master.yml` on ARM64 machines

**Pull request merged**